patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -121,6 +127,17 @@
     public boolean func_219970_i(ItemStack p_219970_1_) {
        return p_219970_1_.func_77973_b() == Items.field_222114_py;
     }
+@@ -306,4 +349,10 @@
++   @Deprecated
+   @Nullable
+   public Food func_219967_s() {
+      return this.field_219974_q;
+   }
++
++   @Nullable
++   public Food getFood(ItemStack stack) {
++      return this.field_219974_q;
++   }
```

Comment:
a. New methods like this should be in IForgeItem
b. This is a breaking change as if any mod overrides `getFood` this then doesn't gracefully fallback to that override